cirq-core/cirq/sim/sparse_simulator_test.py

```diff
@@ -230,51 +255,61 @@ def test_run_param_resolver(dtype):
             np.testing.assert_equal(result.params, param_resolver)
 
 
-@pytest.mark.parametrize('dtype', [np.complex64, np.complex128])
-def test_run_mixture(dtype):
+@pytest.mark.parametrize(
+    'dtype,split', itertools.product([np.complex64, np.complex128], [True, False])
+)
+def test_run_mixture(dtype: Type[np.number], split: bool):
     q0 = cirq.LineQubit(0)
-    simulator = cirq.Simulator(dtype=dtype)
+    simulator = cirq.Simulator(dtype=dtype, split_untangled_states=split)
     circuit = cirq.Circuit(cirq.bit_flip(0.5)(q0), cirq.measure(q0))
     result = simulator.run(circuit, repetitions=100)
-    assert 20 < sum(result.measurements['0'])[0] < 80
+    assert 20 < sum(result.measurements['0'])[0] < 80  # type: ignore
```

Comment:
[No change needed] Why is `# type: ignore` needed here and below?

Reply:
No idea but mypy started complaining. I'll try a cast to int instead.